### ISSUE TITLE
`package.json` `main` field points to `src` instead of `dist`

### ISSUE THREAD

**Issue by jranby** [marekdedic/imagelightbox#513]
Hi - I am using NPM latest release via webpack and the src folder doesn't exist in node_modules folder? This was working with no change on previous projects, am I missing something?

**Comment by marekdedic** [comment]
Hi,
you're not missing anything - with a change of the build infrastructure, the `src` folder is no longer present - please use files from the `dist` folder - there are both minified and unminified versions there.

**Comment by jranby** [comment]
But the package.json in the NPM 1.2.0 is referencing the entry point main file with the `src` folder in the path not the `dist` folder?

**Comment by jranby** [comment]
```
{
  "name": "imagelightbox",
  "version": "1.2.0",
  "description": "Image Lightbox, Responsive and Touch‑friendly",
  "homepage": "https://marekdedic.github.io/imagelightbox",
  "license": "MIT",
  "repository": "https://github.com/marekdedic/imagelightbox.git",
  "main": "./src/imagelightbox.js",
  "style": "./src/imagelightbox.css",
  "types": "./types/imagelightbox.d.ts",
  "files": [
    "dist",
    "docs",
    "types",
    "CHANGELOG.md"
  ],
  "dependencies": {
    "jquery": ">=1.12.4 <4.0.0"
  },
  "devDependencies": {
    "@types/jquery": "^3.3.31",
    "@typescript-eslint/eslint-plugin": "^5.32.0",
    "@typescript-eslint/parser": "^5.32.0",
    "chai": "^4.2.0",
    "csslint": "^1.0.5",
    "eslint": "^8.21.0",
    "eslint-config-prettier": "^8.5.0",
    "eslint-plugin-compat": "^4.0.2",
    "eslint-plugin-deprecation": "^1.3.2",
    "eslint-plugin-prettier": "^4.2.1",
    "express": "^4.17.1",
    "gulp": "^4.0.2",
    "gulp-autoprefixer": "^8.0.0",
    "gulp-clean-css": "^4.2.0",
    "gulp-concat": "^2.6.1",
    "gulp-connect": "^5.7.0",
    "gulp-rename": "^2.0.0",
    "gulp-typescript": "^5.0.1",
    "gulp-uglify": "^3.0.2",
    "mocha": "^10.0.0",
    "npm-run-all": "^4.1.5",
    "puppeteer": "^16.0.0",
    "stylelint": "^14.9.1",
    "stylelint-no-unsupported-browser-features": "^5.0.3",
    "typescript": "^4.7.4"
  },
  "keywords": [
    "jquery",
    "lightbox"
  ],
  "scripts": {
    "build": "gulp",
    "start": "gulp dev",
    "test": "mocha --recursive test/src",
    "lint:css:csslint": "csslint src/imagelightbox.css",
    "lint:css:stylelint": "stylelint --color 'src/**/*.css'",
    "lint:css": "run-p -c --aggregate-output lint:css:*",
    "lint:ts:eslint": "eslint --color 'src/**/*.ts' 'gulpfile.js'",
    "lint:ts": "run-p -c --aggregate-output lint:ts:*",
    "lint": "run-p -c --aggregate-output lint:*",
    "prepack": "npm run build"
  },
  "browserslist": [
    "cover 95%",
    "not < 0.1%",
    "Firefox ESR",
    "IE >= 9",
    "Android >= 2.3",
    "not OperaMini all"
  ]
}

```

**Comment by marekdedic** [comment]
Oh, then that's a bug in the `main` field, thanks for spotting that!

**Comment by marekdedic** [comment]
Should be fixed in 2.0
